agent/stats/engine.go

```diff
@@ -604,26 +604,34 @@ func (engine *DockerStatsEngine) taskContainerMetricsUnsafe(taskArn string) ([]*
 			continue
 		}
 
-		networkStatsSet, err := container.statsQueue.GetNetworkStatsSet()
+		containerMetric := &ecstcs.ContainerMetric{
+			ContainerName:  &container.containerMetadata.Name,
+			CpuStatsSet:    cpuStatsSet,
+			MemoryStatsSet: memoryStatsSet,
+		}
+
+		task, err := engine.resolver.ResolveTask(dockerID)
 		if err != nil {
-			// we log the error and still continue to publish cpu, memory stats
-			seelog.Warnf("Error getting network stats: %v, container: %v", err, dockerID)
+			seelog.Warnf("Task not found for container ID: %s", dockerID)
+		} else {
+			// send network stats for default/bridge/nat network modes
+			if task.ENI == nil && container.containerMetadata.NetworkMode != "host" {
```

Comment:
just wondering, do you need to deal with linux's "none" network mode?

Reply:
yes, will add it